06_puppy/kasunfdo/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	store := MapStore{}
+	puppy := Puppy{ID: 1, Breed: "Husky", Colour: "White", Value: 4999.98}
+
+	if store.CreatePuppy(&puppy) {
+		fmt.Fprintln(out, "Puppy added to store")
+	}
+}
```

Comment:
Was wondering does this lab even need a main function? Many people seem to have a main function but it doesn't seem necessary based on the lab instructions

Reply:
@nicko-lee in the [requirements](https://github.com/anz-bank/go-course/blob/master/06_puppy/README.md) it says: 
> Create an executable go program in directory 06_puppy/USERNAME (see hints)

So the answer is yes :)